packages/api-reference/src/hooks/useSidebar.ts

```diff
@@ -256,9 +256,48 @@ export function useSidebar(options?: { parsedSpec: Spec }) {
     )
   }
 
+  /**
+   * Flattened list of all entries
+   */
+  const flattenedEntries = computed(() => {
+    return items.value.entries.reduce((acc, entry) => {
+      if (entry.children) {
+        return [...acc, entry, ...entry.children]
+      }
+
+      return [...acc, entry]
```

Comment:
Will improve efficiency considerably with large arrays
```suggestion
      acc.push(entry, ...entry.children || [])
      return acc
```

Reply:
I think this code could add some empty arrays to the flattened list, but I’ll add a variation of it. ✌️